### PR TITLE
Maya: Update Screenshots

### PR DESCRIPTION
**Pull request by MustafaJafar**
# Changelog Description

Update screenshots on these pages as well as using a better location to save these images.
- [x] `docs/addon_maya_artist` 
- [x] `docs/addon_maya_arnold_artist` 
- [x] `docs/addon_maya_admin`

For consistency, I used updated the location of images on these pages:
- [x] `docs/addon_maya_redshift_artist`
- [x] `docs/addon_maya_vray_artist`
- [x] `docs/addon_maya_multiverse_artist`
- [x] `docs/addon_maya_yeti_artist`

### CODE REVIEW

File: website/docs/addon_maya_artist.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ between studios and projects but by default your geometry has to be named proper
 For example `sphere_GEO` or `cube1_GEO`. Geometry needs to have freezed transformations
 and must reside under one group, for example `model_GRP`.
 
-![Model example](assets/maya-model_hierarchy_example.jpg)
+![Model example](assets/maya/artist/maya-model_hierarchy_example.jpg)
```
Comment (tokejepsen) [inline website/docs/addon_maya_artist.md:42]: if we have a subfolder of `maya`, should we remove `maya` from the file name?

Comment (MustafaJafar) [reply website/docs/addon_maya_artist.md:42]: That's fair!

File: website/docs/addon_maya_arnold_artist.md [hunk 2]
```diff
@@ -21,20 +21,20 @@ When creating the instance, two objectsets are created; `content` and `proxy`. M
 In order to utilize operators and proxies, the content and proxy nodes need to share the same names (including the shape names
 and CBIDs). This is done by parenting the content and proxy nodes into separate groups. For example:
 
-![Arnold Scene Source](assets/maya-arnold_scene_source.png)
+![Arnold Scene Source](assets/maya/arnold_artist/arnold_scene_source.png)
```
Comment (tokejepsen) [inline website/docs/addon_maya_arnold_artist.md:24]: I think splitting the images into these workflow folder might be too much. Having `admin` and `artist` would be enough and it pretty clear where images should go.

**Review by tokejepsen — CHANGES_REQUESTED** [review]

**Review by tokejepsen — APPROVED** [review]
Looks good to me.

**Comment by MustafaJafar** [comment]
Do we need a second opinion before clicking merge pull request ?

**Comment by tokejepsen** [comment]
> Do we need a second opinion before clicking merge pull request ?

Yeah, need another approval before we'll merge. Second approver usually merges.

**Pull request merged**